Owlcat/FitsTool.py

```diff
@@ -344,6 +348,11 @@ def main():
                        keep_old=not options.delete_files)
         sys.exit(0)
 
+    # print "%d input image(s): %s"%(len(imagenames),", ".join(imagenames))
+    images = [pyfits.open(img) for img in imagenames]
```

Comment:
This opens all the images - I thought the idea was to keep the number of files low. Should the set not be lazily opened as needed below

Reply:
It's only the stack operations that take a mass of images, and they are dealt with above. By the time you drop down to here, you're doing either a single-image op, or binary arithmetic. So easier to open both images, since the code below expects it that way.